tests/page-objects/default/sms/messages.wdio.page.js

```diff
@@ -65,22 +96,51 @@ const getMessageLoadingStatus = async () => {
   return await (await messagesLoadingStatus()).getText();
 };
 
+const sendReply = async (message) => {
+  await (await replyMessage()).setValue(message);
+  await (await replyMessageActions()).waitForExist();
+  await (await submitReplyBtn()).waitForClickable();
+  await (await submitReplyBtn()).click();
+};
+
+const replyAddRecipients = async (message) => {
+  await (await replyMessage()).setValue(message);
+  await (await replyMessageActions()).waitForExist();
+  await (await addRecipient()).waitForClickable();
+  await (await addRecipient()).click();
+  await (await $(SEND_MESSAGE_MODAL)).waitForDisplayed();
+};
+
+const getAmountOfMessages = async () => {
+  await (await $(MESSAGE_CONTENT)).waitForDisplayed();
+  const listedMessages = (await messages());
+  return listedMessages.length;
+};
+
+const getMessagesModalDetails = async () => {
+  await (await $(SEND_MESSAGE_MODAL)).waitForDisplayed();
+  return {
+    recipient: await $(`${SEND_MESSAGE_MODAL} .select2-selection__choice`).getText(),
+    message: await messageText().getAttribute('ng-reflect-model'),
+  };
+};
+
 module.exports = {
+  //messageByIndex,
+  openMessage,
+  getMessageInListDetails,
+  getMessageHeader,
+  getMessageContent,
+  sendMessage,
+  sendReplyNewRecipient,
+  sendMessageToContact,
   exportMessages,
-  messageByIndex,
-  waitForMessagesInLHS,
-  listMessageHeading,
-  listMessageSummary,
-  listMessageLineage,
-  clickLhsEntry,
   messageDetailsHeader,
-  messageContentText,
-  messageContentIndex,
-  messageDetailStatus,
   messagesList,
   messageText,
-  submitMessage,
-  searchSelect,
   getMessageLoadingStatus,
-  sendMessageToPhone,
+  sendReply,
+  replyAddRecipients,
+  getAmountOfMessages,
```

Comment:
The only place I see the `messagesList` function getting used is in send-message.wdio-spec.js where it is used to assert the number of messages.  Is this `getAmountOfMessages` function essentially returning the same count?  Might be missing something here, but if these are basically the same, we can probably get rid of one of them.

Reply:
Ahhh, got it!  Yeah, adding some mention of left/right might be helpful here!